MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,28 +1,86 @@
     private var correctAnswers: Int = 0
     
+    private let questionsAmount: Int = 10
+    private var questionFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
+    private var alertPresenter: AlertPresenter?
+    private var statisticService: StatisticService?
+    
     // MARK: - Lifecycle
     
     override func viewDidLoad() {
         super.viewDidLoad()
         imageView.layer.masksToBounds = true
         imageView.layer.cornerRadius = 20
-        let currentQuestion = questions[currentQuestionIndex]
-        let currentViewModel = convert(model: currentQuestion)
-        show(quiz: currentViewModel)
+        questionFactory = QuestionFactory(delegate: self)
+        questionFactory?.requestNextQuestion()
+        alertPresenter = AlertPresenter(delegate: self)
+        statisticService = StatisticServiceImplementation()
+        
+        let tmp = FileManager.default
+        var tmpURL = tmp.temporaryDirectory
+        let filePi = "pi.txt"
+        tmpURL.appendPathComponent(filePi)
+        let pi = "\(Double.pi)"
+        let dataPi = pi.data(using: .utf8)
+        tmp.createFile(atPath: tmpURL.path, contents: dataPi)
+        
+        //        var jsonString = ""
+        //        let jsonStringURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+        //        if let jsonStringReader = jsonStringURL?.appendingPathComponent("inception.json") {
+        //            do {
+        //                jsonString = try String(contentsOf: jsonStringReader, encoding: .utf8)
+        //            } catch {
+        //                print("Reading error")
+        //            }
+        //        }
+        
+        var jsonQuizString = ""
+        let jsonQuizStringURL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first
+        if let jsonQuizStringReader = jsonQuizStringURL?.appendingPathComponent("top250MoviesIMDB.json") {
+            do {
+                jsonQuizString = try String(contentsOf: jsonQuizStringReader, encoding: .utf8)
+            } catch {
+                print("Reading error")
+            }
+        }
+        
+        let data = jsonQuizString.data(using: .utf8)!
         
+        do {
+            let movieItems = try JSONDecoder().decode(MovieItems.self, from: data)
+        } catch {
+            print("Failed to parse: \(error.localizedDescription)")
+        }
+    }
+    // MARK: - QuestionFactoryDelegate
+    func didRecieveNextQuestion(question: QuizQuestion?) {
+        guard let question = question else {
+            return
+        }
+        
+        currentQuestion = question
+        let viewModel = convert(model: question)
+        DispatchQueue.main.async { [weak self] in
+            self?.show(quiz: viewModel)
+        }
     }
     
+    // MARK: - AlertDelegate
+    func show(alert: UIAlertController) {
+        self.present(alert, animated: true, completion: nil)
```

Comment:
Если возможно, рекомендую использовать самую короткую форму синтаксиса
```suggestion
        present(alert, animated: true, completion: nil)
```